server/etc/pulp/server.conf

```diff
@@ -164,7 +163,6 @@
 
 [data_reaping]
 # reaper_interval: 0.25
-# archived_calls: 0.5
```

Comment:
This can't be removed due to semantic versioning applied to the config files. It needs a deprecated comment added after it. See this line example:  https://github.com/shubham90/pulp/blob/i952/server/etc/pulp/server.conf#L120